test/libethereum/StateTestsFiller/Homestead/stDelegatecallTestFiller.json

```diff
@@ -1290,5 +1302,397 @@
             "to" : "095e7baea6a6c7c4c2dfeb977efac326af552d87",
             "value" : "100000"
         }
-    }
-}
+   },
+
+      "delegatecallEmptycontract" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "45678256",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1000000",
+            "currentTimestamp" : "1",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+	"expect" : {
+		"1000000000000000000000000000000000000000" : {
+		    "storage" : {
+			"0x00" : "0x01"
+		    }
+		}
+        },
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+	    "1000000000000000000000000000000000000000" : {
+                "balance" : "1000",
+                "code" : "{ [[ 0 ]] (DELEGATECALL 50000 0x945304eb96065b2a98b57a48a06ae28d285a71b5 0 64 0 64 )}",
+                "nonce" : "0",
+                "storage" : {
+                }
+            }
+        },
+        "transaction" :
+        {
+            "data" : "",
+            "gasLimit" : "105044",
+            "gasPrice" : "1",
+            "nonce" : "0",
+            "secretKey" : "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8",
+            "to" : "1000000000000000000000000000000000000000",
+            "value" : ""
+        }
+    },
+
+    "delegatecallInInitcodeToExistingContract" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "45678256",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1000000",
+            "currentTimestamp" : "1",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+	"expect" : {
+	    "13136008b64ff592819b2fa6d43f2835c452020e" : {
+                "balance" : "0x01",
+                "storage" : {
+                    "0x00" : "0x01",
+                    "0x01" : "0x1000000000000000000000000000000000000000",
+                    "0x02" : "0x01",
+		    "0x0b" : "0x1000000000000000000000000000000000000000"
+                }
+            }
+        },
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "10000000000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+
+	    "1000000000000000000000000000000000000000" : {
+                "balance" : "10000",
+		"nonce" : "0",
+                "code" : "{ [[ 0 ]] (DELEGATECALL 100000 0x945304eb96065b2a98b57a48a06ae28d285a71b5 0 64 0 64 ) [[ 1 ]] (CALLER) }",
```

Comment:
this is a filler. first two "code" keys are source information needed to modify the contract code if something to change. the actual test file would read and output only las key as it would be redefinition of all the previous one's